content/concepts/introduction.md

```diff
@@ -11,7 +11,7 @@ Ocean Protocol provides tools for developers to _build data markets_, and to _ma
 
 **Build Data Markets.** Use Ocean Protocol software tools to build your own data marketplace, by either forking [Ocean Market](https://market.oceanprotocol.com/) code or building up with Ocean components.
 
-**Manage Datatokens for use in DeFi.** Use Ocean [JavaScript](https://github.com/oceanprotocol/ocean-lib-js) or [Python](https://github.com/oceanprotocol/squid-py) drivers to manage datatokens:
+**Manage Datatokens for use in DeFi.** Use Ocean [JavaScript](https://github.com/oceanprotocol/ocean-lib-js) or [Python](https://github.com/oceanprotocol/ocean.py) drivers to manage datatokens:
```

Comment:
Thanks, good find, let's make it complete :-) 

```suggestion
**Manage Datatokens for use in DeFi.** Use Ocean [JavaScript](https://github.com/oceanprotocol/ocean.js) or [Python](https://github.com/oceanprotocol/ocean.py) drivers to manage datatokens:
```

Reply:
Nice catch, I didn't notice this redirect